frontend/packages/kubevirt-plugin/src/components/modals/modal/modal-footer.tsx

```diff
@@ -55,7 +59,9 @@ type ModalFooterProps = {
 
 export const ModalFooter: React.FC<ModalFooterProps> = ({
   id,
+  className = 'co-m-btn-bar modal-footer kubevirt-create-nic-modal__buttons',
   errorMessage = null,
```

Comment:
why all the `null` and `false` values as defaults ? you can keep the these props empty and the result will be the same and the function declaration more tidy